src/devices/DCMotor/DCMotor.cs

```diff
@@ -189,7 +211,7 @@ public static DCMotor Create(PwmChannel speedControlChannel, int directionPin, i
         /// </summary>
         /// <param name="speedControlPin">Pin used to control the speed of the motor with software PWM (frequency will default to 50Hz)</param>
         /// <param name="directionPin">Pin used to control the direction of the motor</param>
-        /// <param name="otherDirectionPin">Pin used to control the direction of the motor</param>
+        /// <param name="otherDirectionPin">Pin used to control the reverse direction of the motor</param>
```

Comment:
this is actually not correct, original text was. Both pins control the direction, there is not really good description for them because they control the voltage.

Reply:
Ok. The problem here was that StyleCop complains that "Documentation should not be copy-pasted", which happens when two parameters have the same description (which, in most cases, really points to an error). I'll change it to "first pin...", "Second pin..." then.